### ISSUE TITLE
Tag a new release to provide a published docker image

### ISSUE THREAD

**Issue by exhuma** [psf/httpbin#23]
With the recent changes, would it be possible to get a release tagged to make the official docker-image available?

We're planning to deploy a container based on this image internally, and sitting behind a corporate firewall makes it fiddly to build the image due to issues with pip downloads. A docker image would pass through our vetted docker proxy and would simplify this a lot.

**Comment by kevin1024** [comment]
Yep! I'll see what I can do but may ask you for help if I get stuck :)

**Comment by kevin1024** [comment]
Welp, something's weird: https://github.com/psf/httpbin/actions/runs/5985175736.  SOS @exhuma :)

**Comment by exhuma** [comment]
I'm on it. Not sure what's broken considering that it works for some jobs but fails for others. I'll look into it.

**Comment by exhuma** [comment]
The PR above might fix this. I hope. The comments of the PR should explain the details.

**Comment by exhuma** [comment]
@kevin1024 did you get the chance to look at the PR?

**Comment by kevin1024** [comment]
Merged. I'll see if i can trigger a release again.  Maybe I can untag and retag?

**Comment by exhuma** [comment]
Considering that there was an issue with the actual release job, the tagged code never ended up in an official package. I would go for it. Just force-tag and force-push.

**Comment by kevin1024** [comment]
I think it actually made a release?  https://pypi.org/project/httpbin/#history

**Comment by kevin1024** [comment]
I ran the release job a couple of times, I think the second time was the charm?

**Comment by exhuma** [comment]
That's weird. On [run #3](https://github.com/psf/httpbin/actions/runs/6005769003) the job was skipped. On [run #4](https://github.com/psf/httpbin/actions/runs/6005774902) it was not. The skip happens if one of the `if` rules returns `false`. For example this one: https://github.com/psf/httpbin/blob/9a3dd8905d5bbaee9d170fc7e2c79648a5d608c5/.github/workflows/publish.yml#L83

Everything does seem fine to me. I don't understand why one of them was skipped.

**Comment by kevin1024** [comment]
Right on.  I think we are good for the current release then, right?  I'll go ahead and close this issue.  Thanks so much for your work on this!

**Comment by exhuma** [comment]
You're welcome.

I noticed yesterday that the docker image does not show up on the public github page. But the workflow job did complete successfully. Could this be a permission issue for the container registry? I found [a section in the docs explaining this](https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility#about-setting-visibility-and-access-permissions-for-packages). In case you see it on your own machine, try opening the page in incognito-mode to verify. Normally, on the right hand side on github it should show up under releases with a container icon like this:

![image](https://github.com/psf/httpbin/assets/65717/b0990c97-c3b3-4ef3-a7b5-b18976c5acc2)


If we can't get this to work we could also push it to docker-hub. The change in the CI is negligible. The github registry has the advantage that authorisation is easier though.

Another way to test it is to run `docker pull ghcr.io/psf/httpbin:0.10.1` without logging in. This has to work for people to use the image.
